frontend/packages/core/src/AppLayout/index.tsx

```diff
@@ -1,16 +1,34 @@
 import React from "react";
 import { Outlet } from "react-router-dom";
+import styled from "@emotion/styled";
+import { Box as MuiBox } from "@material-ui/core";
 
+import Drawer from "./drawer";
 import FeedbackButton from "./feedback";
 import Header from "./header";
 
-const AppLayout: React.FC = ({ children }) => {
+const Box = styled(MuiBox)({
```

Comment:
based this off a material ui drawer example https://material-ui.com/components/drawers/#mini-variant-drawer, sandbox. and for `width: "calc(100% - 100px)`, pattern matched off AIM